.github/workflows/bump-version.yml

```diff
@@ -0,0 +1,244 @@
+name: Bump version
```

Comment:
This is the workflow `pyhf` uses to bump the version in CI so that we don't accidentally ever commit something wrong.